### ISSUE TITLE
Button A11y: text is not visible in High Contrast Black Mode while navigating using Tab key

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10778]
[Accessibility][Supporting the Platform : Panel Controls ] : Name of the control is not visible in High Contrast Black Mode while navigating by using Tab key.

**User Impact:**
All users will be impacted if in high contrast black mode  control name is not visible when focus lands on control.

**Usage:**
All Usage.

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 18999.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/panel
Edge Browser Version:
Microsoft Edge: 44.18999.1.0
Microsoft EdgeHTML: 18.18999

**Repro Steps:**
1. Turn on the high Contrast Mode ( Black / White ).​​
2. Open https://developer.microsoft.com/en-us/fabric#/controls/web/panel in Edge.
3. Navigate by using Tab key.
4. Verify the issue when focus lands on 'Open Panel' button.

**Actual Result:**
While navigating by using Tab key, when keyboard focus land on 'Open Panel' control. 'Open Panel' is not visible in high contrast mode.

**Expected Result:**
Name of the control 'Open Panel' should be properly visible in high contrast black mode.

**Note:**
We also observe that controls are not adapting the high contrast settings. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Comment by ecraig12345** [comment]
This is a general bug with [all DefaultButtons on the website](https://developer.microsoft.com/en-us/fabric#/controls/web/button) (it's not specific to Panel examples) and doesn't repro in [a codepen](https://codepen.io/ecraig12345/pen/wvvKmVr?&editable=true) or for PrimaryButtons. So there must be something weird about either our website's styles or the UHF styles which messes up the buttons when focus is visible in high contrast.

**Comment by khmakoto** [comment]
There's a lot more discussion about this subject in #8359. But the tldr is that the issue is caused by 3rd party css on the page (UHF styles like Elizabeth mentioned above) so there's not a whole lot we can do about it unless we move to another site.

**Comment by khmakoto** [comment]
@999-Dev I've started a meta issue in #10911 tracking this type of bugs caused by UHF styles and included this one there. Hence I'll be closing this bug now. Thanks!

**Comment by ecraig12345** [comment]
I'm going to reopen this because I think it's worth fixing even if the fix is less universal than we'd like.

**Comment by micahgodbolt** [comment]
![image](https://user-images.githubusercontent.com/1434956/70750589-86074a00-1ce3-11ea-9a88-4241a184408c.png)

We've recently made improvements to buttons in high contrast. Seems there is some odd boxing going on due to the host site styles, but the button is now readable in high contrast mode.
